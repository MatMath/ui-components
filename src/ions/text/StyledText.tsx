import styled from 'styled-components';

export const StyledText = styled.span`
  font-family: inherit;
  font-weight: 600;
`;
