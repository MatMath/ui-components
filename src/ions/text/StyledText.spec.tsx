import * as React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import { StyledText } from './StyledText';

describe('<Text>', () => {
  describe('global render', () => {
    it('should render', () => {
      const TextWrapper = mount(<StyledText />);

      expect(TextWrapper).toMatchSnapshot();
    });
  });
});
