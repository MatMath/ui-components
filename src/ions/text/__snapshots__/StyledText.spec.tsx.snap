// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text> global render should render 1`] = `
.c0 {
  font-family: 'Source Sans Pro',Helvetica,Arial,sans-serif;
  font-weight: 500;
  line-height: 1.125rem;
  font-size: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<styled.span>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "
  ",
            "
  font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif;
",
            "
  font-weight: 500;
  line-height: 1.125rem;
  font-size: ",
            [Function],
            ";
  display: flex;
",
          ],
        },
        "displayName": "styled.span",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "span",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <span
      className="c0"
    />
  </StyledComponent>
</styled.span>
`;
